pkg/storage/segments/store.go

```
@@ -203,6 +210,15 @@ func (s *segmentStore) Get(ctx context.Context, path paths.Path) (
 	return rr, convertMeta(pr), nil
 }
 
+func makeErasureScheme(minReq, total, eShareSize int32) (eestream.ErasureScheme, error) {
+	fc, err := infectious.NewFEC(int(minReq), int(total))
+	if err != nil {
+		return nil, err
```

Should we return a wrapped error here? `Error.Wrap(err)`